rollup-tests.config.js

```diff
@@ -18,6 +18,9 @@ export default {
     file: 'build/scripts/tests.bundle.js',
     format: 'es',
     sourcemap: true,
+
+    // Simplify build output by putting all code in one bundle.
+    inlineDynamicImports: true,
```

Comment:
As the name suggests, this causes code imported via `import(...)` to be pulled directly into the bundle that imports it.